docs/src/main/paradox/scala/http/release-notes/10.1.x.md

```diff
@@ -2,13 +2,82 @@
 
 ## 10.1.0-RC1
 
-### Akka is not an explicit dependency any more
+See the [announcement](https://akka.io/blog/news/2017/12/22/akka-http-10.1.0-RC1-released.html) and
+closed tickets on the [10.1.0-RC1 milestone](https://github.com/akka/akka-http/milestone/26?closed=1).
 
-Akka HTTP supports both running on top of Akka 2.4 and Akka 2.5. However, using Akka HTTP with Akka 2.5 used to be
+10.1.0-RC1 is the first release candidate for the next minor version of Akka HTTP. The most important changes are
+
+ * Removing support for Akka 2.4.x which is at its end-of-life with the end of 2017. In the future, this will allow us
+   to make use of features that only Akka 2.5 supports.
+ * Removing methods that have been deprecated during the life-time of Akka HTTP 10.0.x. For now, methods only deprecated
+   in the last release of Akka HTTP, 10.0.11, are not yet removed to allow for a smooth transition.
+ * The new client pool implementation introduced in 10.0.11 is now the default.
```

Comment:
Please try it out and provide feedback during the RC period of this release so we can iron out any problems we may have missed initially.